### ISSUE TITLE
Question: Plugin.xml AdminLinks

### ISSUE THREAD

**Issue by LaocheXe** [e107inc/e107#2926]
Currently I tried doing something like this:

	<adminLinks>
		<link url="admin_config.php" description="" icon="" iconSmall="" icon128="" primary="true" >LAN_CONFIGURE</link>
		<link url="admin_application.php" description="" icon="" iconSmall="" icon128="" primary="false" >Applications</link>
		<link url="admin_sr.php" description="" icon="" iconSmall="" icon128="" primary="false" >Service Records</link>
		<link url="admin_roster.php" description="" icon="" iconSmall="" icon128="" primary="false" >Roster Management</link>
		<link url="admin_rs_logs.php" description="" icon="" inconSmall="" icon128="" primary="false" >Roster Logs</link>
		<link url="admin_cshops.php" description="" icon="" inconSmall="" icon128="" primary="false" >C-Shops</link>
	</adminLinks>

thinking it would add more "plugin icons/links" in the plugin section in the admin area - which didnt work, I guess it checks for the first line, and leaves the rest alone??

**Comment by rica-carv** [comment]
Perhaps. That section is also rendered in the drop down menu in admin area, so i think it will only allow one link, and ignores the remaining ones...

**Comment by LaocheXe** [comment]
Very odd, checked the drop down menu in admin area and don't see any of the links that are in the plugin.xml

**Comment by CaMer0n** [comment]
@LaocheXe Yes, and prevents plugins from over running the admin.  Imagine if every plugin did that.. it would look like chaos in the admin area.

**Comment by LaocheXe** [comment]
@CaMer0n I did this as a test after reading the plugin.xml, as it's says <adminLinks> which sould be <adminLInk> since there should only be one, correct? Or did you have future plains for this? Maybe a central plugin section. Example, since the first link is set to primary, once the admin goes into that plugin's page, maybe have a box section of icon links to other pages at the bottom of the page, that looks similar to the Plugins section in admin area?

**Comment by CaMer0n** [comment]
@LaocheXe If I understand well, you're looking for something similar to the "Database" page as a default plugin page, correct?

**Comment by LaocheXe** [comment]
@CaMer0n Similar, more compact in a way.

As Database is large:
![capture1](https://user-images.githubusercontent.com/5818022/34179881-43d0ad98-e4d2-11e7-80ce-34dddc0f79a5.JPG)

I was thinking another approach - something like the plugin section:
![capture2](https://user-images.githubusercontent.com/5818022/34179908-5707fe02-e4d2-11e7-970b-7bc46ebc27a1.JPG)


Because right now, this looks a little ugly, and trust me, I haven't added any more, but all the database tables requires, I would add another 10 or more links to this:
![capture3](https://user-images.githubusercontent.com/5818022/34179943-7eda295a-e4d2-11e7-8a39-013e9692e384.JPG)


But how I would like to see/imagine is something similar to this:
![capture3-edit](https://user-images.githubusercontent.com/5818022/34179963-93f47d18-e4d2-11e7-9bff-162799daa8fb.png)

**Comment by Jimmi08** [comment]
@LaocheXe from the same reason I moved all create links to right panel

![image](https://user-images.githubusercontent.com/5429548/34180115-cf61c94c-e50d-11e7-8280-1b6ae5360915.png)

**Comment by CaMer0n** [comment]
@LaocheXe I understand. Do you have this plugin in a Github repo already?

**Comment by LaocheXe** [comment]
@CaMer0n yes I do, but the last image I uploaded was edit - I don't really have that, plus still work in progress, a lot of generated pages that I plain on using. https://github.com/LaocheXe/RSystems

**Comment by nlstart** [comment]
Would be nice if <adminLinks> section of each plugin.xml would support a toggle functionality to hide/unhide underlying menu items. Also a separator link like `<li class='divider' role='separator'></li>` should be helpful within the <adminLinks> section.

**Comment by Jimmi08** [comment]
Supporting to solve this somehow in plugin.xml.  There is already the attribute primary for links.  
Either toggle or 3rd level menu or generated dashboard. 
For now, I need to create a new plugin for each option...
